### DISCUSSION TITLE
How to block clients using doh but do not have a unique name?

### DISCUSSION THREAD

**Discussion by sddzcuigc** [AdguardTeam/AdGuardHome#3800] [Q&A]
I have a question , I need to block some clients without name,because they join my dns without my accepttion , they use the doh without any client name, I asked them to join with the addons like"https://my_dns_name/dns-quesy/client1 "   but they use https:// my dns name/dns-query/ how can I block them?

**Comment by ainar-g** [comment]
You can block clients by IP and subnet in the DNS settings.
